e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -162,9 +165,9 @@ private void loadDockerComposeContainers(int apiPort) {
                 .withLocalCompose(true)
                 .withScaledService("worker", 2)
                 .withExposedService("api", DEFAULT_API_PORT, new HostPortWaitStrategy()
-                    .withStartupTimeout(Duration.of(200, SECONDS)));
-        //.withLogConsumer("worker", new Slf4jLogConsumer(log)) // Use to debug, for now there's too much log data
-        //.withLogConsumer("api", new Slf4jLogConsumer(log));
+                    .withStartupTimeout(Duration.of(200, SECONDS)))
+        .withLogConsumer("worker", new Slf4jLogConsumer(log)) // Use to debug, for now there's too much log data
```

Comment:
Put a todo in so that we turn it off later to avoid too much logging.

Reply:
Removed